riscv/src/register/satp.rs

```diff
@@ -114,3 +138,25 @@ pub unsafe fn set(mode: Mode, asid: usize, ppn: usize) {
     let bits = (mode as usize) << 60 | (asid << 44) | ppn;
     _write(bits);
 }
+
+/// Sets the register to corresponding page table mode, physical page number and address space id.
```

Comment:
```suggestion
/// Attempts to set the register to corresponding page table mode, physical page number and address space id.
```

Missed the update to the docs, will add with next set of changes.